music_assistant/controllers/player_queues.py

```diff
@@ -2084,6 +2084,7 @@ def _handle_playback_progress_report(
             EventType.MEDIA_ITEM_PLAYED,
             object_id=item_to_report.media_item.uri,
             data=MediaItemPlaybackProgressReport(
+                item_id=item_to_report.media_item.item_id,
```

Comment:
why is this needed ? There is alrerady an uri reported

Reply:
I've removed the changes in player_queues, parsing the provided path now.